README.md

```diff
@@ -3,42 +3,50 @@
 This repo serves to backup my neovim config file. I ended up making this public
 again for convenience.
 
-## Installation.
-Get Vim Plug (https://github.com/junegunn/vim-plug)
+## Installation
+
+First install Neovim. For Ubuntu, run the following:
+```
+sudo add-apt-repository ppa:neovim-ppa/stable
+sudo apt-get install -y neovim
+```
+
+Next, get Vim Plug (https://github.com/junegunn/vim-plug) by running the following:
 ```
 sh -c 'curl -fLo "${XDG_DATA_HOME:-$HOME/.local/share}"/nvim/site/autoload/plug.vim --create-dirs \
        https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 ```
-Afterwards, simply copy and paste the init.vim file into `~/.config/nvim/init.vim`.
+Afterwards, make a `nvim` directory under `~/.config/` and copy and paste the `init.vim` file from this repository into `~/.config/nvim/init.vim`.
 
-Then just install the plugins inside neovim using:
+Now install the plugins inside Neovim by opening `init.vim` with `nvim` and using
 ```
 :PlugInstall
 ```
-Afterwards, the plugins will install. Afterwards, quit and neovim should be configured.
+Afterwards, quit and Neovim should be configured.
 
 Vim support is likely indentical but I haven't tested it on Vim (I have a few
-neovim specific plugins for lsp, treesitter and autocompletion support)
+Neovim specific plugins for LSP, treesitter and autocompletion support)
 
-To use the fixers and linters, you should install the relevant linters + fixers
+To use the fixers and linters, you should install the relevant linters and fixers
 (see the config file for the relevant ones). You also will need to install any
-LSPs that you want.
+LSPs that you want. For Python, use pip3 to install the following: pyright,
```

Comment:
Could you put a section for Python instead ? 

I feel like things would get messy if the installation for other langs are added

Reply:
Done